ui/lib/sync/addon/components/secrets/page/destinations/destination/details.hbs

```diff
@@ -6,13 +6,20 @@
 <Secrets::DestinationHeader @destination={{@destination}} />
 
 {{#each @destination.formFields as |field|}}
-  {{#let (get @destination field.name) as |value|}}
+  {{#let (get @destination field.name) as |fieldValue|}}
     {{#if (includes field.name @destination.maskedParams)}}
       <InfoTableRow @label={{or field.options.label (to-label field.name)}}>
-        <Hds::Badge @text={{this.credentialValue value}} @icon="check-circle" @color="success" />
+        <Hds::Badge @text={{this.credentialValue fieldValue}} @icon="check-circle" @color="success" />
       </InfoTableRow>
+    {{else if (eq field.name "customTags")}}
```

Comment:
this could also be `(eq field.type "object")` but it didn't feel necessary to be that abstract at this point